palace/linalg/iterative.cpp

```diff
@@ -646,12 +644,12 @@ void GmresSolver<OperType>::Mult(const VecType &b, VecType &x) const
     }
     else  // B && pc_side == PrecSide::RIGHT
     {
-      r = 0.0;
+      r.VecType::operator=(0.0);
```

Comment:
The `using operator=` in the workspace vector classes should mean these method casts aren't necessary.

Reply:
Resolved above.